packages/client/tests/functional/composites-list/upsert-create.ts

```diff
@@ -1,6 +1,6 @@
 import { faker } from '@faker-js/faker'
 
-import { setupTestSuite } from '../_matrix'
+import { setupTestSuite } from './_matrix'
```

Comment:
These existing files were moved to support the nested test suite approach. Because of the fact that these files were nested in their own dir the test suite thinks it's a new test.

Reply:
> Because of the fact that these files were nested in their own dir the test suite thinks it's a new test
I not sure I understood this bit. Can you please expand/explain again?

Reply:
Please see this https://github.com/prisma/prisma/pull/14831#discussion_r946667582

These files needed to be moved out of their own directory because the changes to the test suite will treat this directory as a new test suite and fail.